src/criros/raveutils.py

```diff
@@ -174,13 +177,29 @@ def get_transform_from_tf(self, parent, child, time=None):
     T[:3,3] = pos
     return T
   
-  def stop(self):
+  def stop(self, delay=0):
     """
+    @type  delay: float
+    @param delay: Delay time(s) before shutting down the update thread.
+                  Suggested value is 0.3 to accomodate delay around 0.2s
+                  in encoder feedback. This ensures openrave env state is
+                  up-to-date and correct before update stops.
     If the update thread is running, stops it.
-    Once you stop the update thread B{you have to call manually} the C{update_rave_environment}.
+    Once you stop the update thread you can either manually call 
+    C{update_rave_environment} or call C{restart} to restart updating.
     """
```

Comment:
Parameter description should go after function description. Avoid extra indentation because the documentation will have additional unwanted indentation.
```{python}
"""
If the update thread is running, stops it.
Once you stop the update thread B{you have to call manually} the C{update_rave_environment}.
Once you stop the update thread you can either manually call 
C{update_rave_environment} or call C{restart} to restart updating.
@type  delay: float
@param delay: Delay time(s) before shutting down the update thread.
Suggested value is 0.3 to accomodate delay around 0.2s in encoder feedback. 
This ensures openrave env state is up-to-date and correct before update stops.
"""
```